src/Components/Results/Results.js

```diff
@@ -0,0 +1,65 @@
+import React, {useState, useEffect} from 'react'
+import Card from '../Card/Card'
+import './Results.css'
+import { searchAlbums } from '../../api'
+
+function Results({query}) {
+  const [results, setResults] = useState([])
+  const [pagination, setPagination] = useState({page: 1, pages: 1})
+  const [isLoading, setLoading] = useState(true)
+
+  useEffect(() => {
+    setLoading(true)
+    searchAlbums(query, 1)
+      .then(data => {
+        setResults(data.results)
+        setPagination(data.pagination)
+        setLoading(false)
+      })
+      .catch(error => alert(error.message))
+  }, [ query ])
+
+  const handleClick = (direction) => {
+    setLoading(true)
+    if (direction === 'last') {
+      searchAlbums(query, pagination.page - 1)
+      .then(data => {
+        setResults(data.results)
+        setPagination(data.pagination)
+        setLoading(false)
+        })
+      .catch(error => alert(error.message))
+    }
+    if (direction === 'next') {
+      searchAlbums(query, pagination.page + 1)
+      .then(data => {
+        setResults(data.results)
+        setPagination(data.pagination)
+        setLoading(false)
+      })
+      .catch(error => alert(error.message))
+    }
+    window.scrollTo(0, 0)
+  }
+
+  return (
+    !isLoading ? (<section className='results'>
+      {results.length ? (
+        <div className='results--grid'>
+            {results.map(result => (
+              <Card key={result.masterId} result={result}/>
+            ))}
+        </div>
+        ) : <div className='results--none'><p>No results for {query}</p></div>}
+      {results.length && <div className='results--pages'>
+        {pagination.page > 1 && <button className='results--last'onClick={() => handleClick('last')}>last</button>}
```

Comment:
Changing last to previous makes more sense to me. When I click the last button I expect to go to the last page. So if there is 1 of 200, clicking last would take me to 200

Reply:
ah that makes sense, changed it

Reply:
nice perspective on the wording 👍